### PR TITLE
release: update instructions

### PR DESCRIPTION
**Pull request by garyverhaegen-da**
CHANGELOG_BEGIN
CHANGELOG_END

### CODE REVIEW

File: release/RELEASE.md [hunk 1]
```diff
@@ -10,14 +10,26 @@ patches we backport to the 1.0 release branch).
...
 
+> **TRANSITION**: We have switched to a new release process that includes
+> Canton. In the new process, a smaller set of artifacts is generated from this
+> repo as a "split" release, then the Canton team needs to make a release based
+> on that, and then the final release is triggered from the [DACH-NY/assembly]
+> repo, referencing both this and the Canton snapshots.
+>
+> Pre-2.0.0 releases should still follow the old process, for which the steps
+> are marked **[1]**. 2.0.0 snapshots and later should follow the new process,
+> for which steps are marked **[2]**.
```
Comment (stefanobaghino-da) [inline release/RELEASE.md:21]: Should have a separate document for this? I suspect that the interlaced instructions could be confusing. The old release instructions are definitely stable, so some duplication can be acceptable here, I believe.

File: release/RELEASE.md [hunk 2]
```diff
@@ -47,13 +59,33 @@ patches we backport to the 1.0 release branch).
...
+
+1. **[2]** Open a PR on the [DACH-NY/assembly] repo using either the Canton
+   version that just got released (for a stable release) or the latest nightly
+   snapshot (for a snapshot). See the instructions in the README of the
+   [DACH-NY/assembly] repo for details.
+
 1. **[STABLE]** The PR **must** be approved by a team lead before merging. As
-   of this writing (2021-10-12), @bame-da, @gerolf-da, @cocreature, @stefanobaghino-da or @adriaanm-da.
+   of this writing (2021-10-12), @bame-da, @gerolf-da, @cocreature,
+   @stefanobaghino-da or @adriaanm-da.
```
Comment (stefanobaghino-da) [inline release/RELEASE.md:88]:
```suggestion
   of this writing (2022-02-08), @bame-da, @gerolf-da, @cocreature,
   @ray-roestenburg-da  or @adriaanm-da.
```

**Comment by cocreature** [comment]
What do we want to do with the release rotation now? I don’t think we can expect people to update Canton & release that and without commitment from the Canton team to coordinate on this weekly, this isn’t going to work with reasonable effort.

**Review by stefanobaghino-da — COMMENTED** [review]
I think it could make sense for @ray-roestenburg-da to review the changes here and approve, given that he's in charge of this tomorrow.

**Comment by garyverhaegen-da** [comment]
> What do we want to do with the release rotation now? I don’t think we can expect people to update Canton & release that and without commitment from the Canton team to coordinate on this weekly, this isn’t going to work with reasonable effort.

The intention (obviously not clear enough) is for these instructions to tell people to just take whatever the latest Canton is at the time of making the weekly snapshot.

I'm trying to coordinate with them to automate on their side so they get "fresh" SDKs reasonably often, but at the very least we'd get a new Canton as they do nightly releases.

File: release/RELEASE.md (hunk 1 above)
Comment (ray-roestenburg-da) [reply release/RELEASE.md:21]: I agree on  the separate docs, it will be easier to follow.

Comment (stefanobaghino-da) [reply release/RELEASE.md:21]: > As in `TESTING.md`, `RELEASE_1.md` and `RELEASE_2.md`?

Something along those lines, yes.

**Comment by garyverhaegen-da** [comment]
I think this is in a bit of a better shape now. @ray-roestenburg-da for tomorrow you'll want to follow `release/RELEASE_SNAPSHOT_2.0.md` then `release/RELEASE.md`.

**Comment by RayRoestenburg** [comment]
Thanks!

**Pull request merged**